stack/lab/Dockerfile

```diff
@@ -16,7 +16,7 @@ WORKDIR /opt/
 # Install additional system packages
 RUN apt-get update --yes && \
      apt-get install --yes --no-install-recommends \
-     curl \
+     curl povray \
```

Comment:
```suggestion
     curl \
     povray \
```

Reply:
And why the `curl` is needed? I think `curl` is useful and cat go to `base`.

Reply:
I am not sure why it is here. I'd rather keep the base basic and slim so keeping it here.